src/dfcx_scrapi/tools/evaluations.py

```diff
@@ -611,3 +760,5 @@ def from_dataframe(self, df: pd.DataFrame) -> pd.DataFrame:
         df = self.validate_and_prep_inputs(df)
 
         return df
+    
```

Comment:
just 1 extra line at the end. linter should have caught this, I'll see about updating it.